src/python/library/tritonclient/http/_client.py

```diff
@@ -198,13 +199,15 @@ def __exit__(self, type, value, traceback):
         self.close()
 
     def __del__(self):
-        self.close()
+        if not self._closed:
+            self.close()
 
     def close(self):
         """Close the client. Any future calls to server
         will result in an Error.
 
         """
```

Comment:
Can you move `if not self._closed` to here?

Reply:
I moved check for double close  to close method. Is it fine?